README.md

```diff
@@ -114,120 +117,126 @@ getting_started=> \d
 
 # Limitations
 
-* No [Git-style CLI](https://docs.dolthub.com/cli-reference/cli) for version control like in [Dolt](https://github.com/dolthub/dolt), only a SQL interface.
-* Can't push to DoltHub or DoltLab, only custom remotes.
-* Limited support of Postgres-specific types and functions.
-* No Postgres system tables.
-* No authentication or users.
-* Database and schema models are merged.
-* Limited support for SSL connections (non-verified connections only).
-* No GSSAPI support.
-* No PostgreSQL functions have been implemented, therefore only MySQL functions may be used.
-* No support for replication, clustering, etc.
+- No [Git-style CLI](https://docs.dolthub.com/cli-reference/cli) for version control like in [Dolt](https://github.com/dolthub/dolt), only a SQL interface.
+- Can't push to DoltHub or DoltLab, only custom remotes.
+- Limited support of Postgres-specific types and functions.
+- No Postgres system tables.
+- No authentication or users.
+- Database and schema models are merged.
+- Limited support for SSL connections (non-verified connections only).
+- No GSSAPI support.
+- No PostgreSQL functions have been implemented, therefore only MySQL functions may be used.
+- No support for replication, clustering, etc.
 
 # Performance
 
 Dolt is [1.7X slower than MySQL](https://docs.dolthub.com/sql-reference/benchmarks/latency) as measured by
 a standard suite of Sysbench tests.
 
-We use these same Sysbench tests to benchmark DoltgreSQL and compare the results to PostgreSQL. 
+We use these same Sysbench tests to benchmark DoltgreSQL and compare the results to PostgreSQL.
 
 Here are the benchmarks for DoltgreSQL version `0.4.0`.
 
 <!-- START_LATENCY_RESULTS_TABLE -->
-|       Read Tests        | PostgreSQL | DoltgreSQL | Multiple |
-|-------------------------|------------|------------|----------|
-| oltp\_point\_select     |       0.13 |       0.54 |      4.2 |
-| oltp\_read\_only        |       2.35 |      12.75 |      5.4 |
-| select\_random\_points  |        0.2 |       1.04 |      5.2 |
-| select\_random\_ranges  |        0.4 |       1.03 |      2.6 |
-| reads\_mean\_multiplier |            |            |      4.4 |
-
-|       Write Tests        | PostgreSQL | DoltgreSQL | Multiple |
-|--------------------------|------------|------------|----------|
-| oltp\_insert             |       0.78 |       3.02 |      3.9 |
-| oltp\_read\_write        |       3.89 |      20.37 |      5.2 |
-| oltp\_update\_index      |       0.81 |       3.19 |      3.9 |
-| oltp\_update\_non\_index |       0.78 |       3.13 |      4.0 |
-| oltp\_write\_only        |       1.37 |       7.56 |      5.5 |
-| writes\_mean\_multiplier |            |            |      4.5 |
+
+| Read Tests            | PostgreSQL | DoltgreSQL | Multiple |
+| --------------------- | ---------- | ---------- | -------- |
+| oltp_point_select     | 0.13       | 0.54       | 4.2      |
```

Comment:
Do these format correctly without the backslashes?

Reply:
Yeah looks like they do. Not seeing a diff there in the rich diff https://github.com/dolthub/doltgresql/pull/801/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5